src/main/java/org/broadinstitute/hellbender/tools/exome/segmentation/CopyRatioSegmenter.java

```diff
@@ -24,27 +25,23 @@
      *                          include in the model.  Hidden states are pruned as the model is learned.
      */
     public CopyRatioSegmenter(final int initialNumStates, final List<SimpleInterval> positions, final List<Double> data) {
-        super(initialNumStates, positions, data);
+        super(initialNumStates, positions, data, initialLog2CopyRatios(initialNumStates));
+        logCoverageCauchyWidth = DEFAULT_INITIAL_CAUCHY_WIDTH;
     }
 
     /**
      * evenly-spaced log-2 copy ratios
      * @param K the initial number of hidden states
      */
-    @Override
-    protected void initializeHiddenStateValues(final int K) {
-        hiddenStateValues = GATKProtectedMathUtils.createEvenlySpacedPoints(-3, 2, K);
-        hiddenStateValues[NEUTRAL_VALUE_INDEX] = NEUTRAL_LOG_2_COPY_RATIO;
+    private static List<Double> initialLog2CopyRatios(final int K) {
+        final List<Double> result = Doubles.asList(GATKProtectedMathUtils.createEvenlySpacedPoints(-3, 2, K));
+        result.set(NEUTRAL_VALUE_INDEX, NEUTRAL_LOG_2_COPY_RATIO);
```

Comment:
Downstream code assumes that the neutral value is at the `NEUTRAL_VALUE_INDEX`, which is by convention never pruned or re-learned in the M step.  While there's not an instant fix, I think I can make this cleaner with a bit of work. . .